dbusmock/templates/logind.py

```diff
@@ -287,3 +290,30 @@ def AddSession(self, session_id, seat, uid, username, active):
     obj_user.Set('org.freedesktop.login1.User', 'Sessions', cur_sessions)
 
     return session_path
+
+@dbus.service.method(SESSION_IFACE,
+                     in_signature='uu',
+                     out_signature='hb')
+def TakeDevice(self, major, minor):
+    sysfs_uevent_path = '/sys/dev/char/{}:{}/uevent'.format(major, minor)
+    sysfs_uevent = open(sysfs_uevent_path, 'r')
+    devname = None
+    for line in sysfs_uevent.readlines():
+        match = re.search('DEVNAME=(.*)', line)
```

Comment:
Let's please be correct here and use `re.match`, just in case there is ever a field like "FOODEVNAME".